Fa/add reading page

felixarntz opened the pull request:
This is a PR to address #18.

Note that either this or #30 will create a merge conflict, depending on which of the two is merged first. This will need to be resolved prior to merging the second.

Commit:
Add Reading settings page.

Commit:
Merge remote-tracking branch 'origin/master' into fa/add-reading-page

Commit:
Merge remote-tracking branch 'origin/master' into fa/add-reading-page

afercia commented:
Everything looks good to me except a couple things:
"Blog pages show at most"
"Syndication feeds show the most recent"
they miss the words "posts" and "items". We should avoid to add them after the selects though. Instead, we should try to reword the labels. For example something like:

"Maximum number of posts to show in the blog pages:"
"Maximum number of most recent items  to show in the syndication feeds:"

or even better wording, the shorter the better. Worth noting having meaningful strings (not split in two parts) would also be an improvement for translations.

Things to consider later:
- remove the link from the label `A static page (select below)`

afercia reviewed (commented):
Commented!

Commit:
Improve field labels in Reading > Blog section.

felixarntz commented:
I changed the labels to the following for now:

* Number of posts to show in blog pages
* Number of most recent items to show in syndication feeds
* Display mode for articles in a feed (I changed this one because prior it also used a sentence where the field value would be part of the sentence, which could, again, cause translation problems)

These labels can probably be improved, but let's use them like that for now.
